pkg/proxy/winkernel/proxier.go

```diff
@@ -96,6 +96,13 @@ type loadBalancerInfo struct {
 	hnsID string
 }
 
+type loadBalancerIdentifier struct {
```

Comment:
add length of endpoints to struct

Reply:
Added endpointsCount to the loadBalancerIdentifier